Kajabi/sage-lib
# refactor(doc site): audit home page for a11y issues

Fixes #720 

## Description

Run an accessibility audit through the home page of the documentation site.


## Testing in `sage-lib`

1. Navigate to [locahost](http://localhost:4000/pages/index)
2. Verify that the home page has no regressions
3. **Important** - confirm the page passes accessibilty audits


## Testing in `kajabi-products`
- N/A - `sage-lib` only
